bundles/org.openhab.binding.gardena/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -592,22 +592,22 @@
 	<channel-type id="duration">
 		<item-type>Number:Time</item-type>
 		<label>Duration</label>
-		<description>Duration in minutes</description>
-		<state readOnly="true" pattern="%d min"/>
+		<description>Duration</description>
+		<state readOnly="true" pattern="%d %unit%"/>
```

Comment:
Is the %unit% declaration here correct? I may be wrong, but my understanding of the new UoM is that the thing xml pattern should specify an actual default unit e.g. 's' => perhaps someone else can comment?

Applies to all similar cases below..

Reply:
As far as I know, it's not the binding's business to set a unit, but it's up to the user to set appropriate metadata. I can dig up a reference if needed.

Reply:
To be more complete on the above: if no unit metadata is set, `%unit%` will use the system default unit (here: seconds), otherwise the metadata specified unit.
We can also keep the pattern as-is (the user can override it anyway), I just found it weird that some channels use `%unit%` and some don't without any clear logic behind it.

Reply:
Yeah. Perhaps the core maintainers can tell us what is the preferred rule?

Reply:
The [respective section of the docs](https://www.openhab.org/docs/developer/bindings/thing-xml.html#state-description) seems to imply the preference is `%unit%` for UoM `Number` channels, but maybe I'm reading too much into it?

Reply:
It depends. If you know the only "good" unit ist `min` use `min`. Otherwise the system unit (which is `s` for time) will be used. 

Whatever you set here has no influence on the way the value is persisted or stored internally in the item, it's solely for displaying. If a user doesn't like what you suggest: the state description metadata takes precedence. You can't do any harm by suggesting a good default. 

If the system unit most likely fits (e.g. distances in `m`/`ft`) you can use `%unit%.

Reply:
Thanks @J-N-K - sounds like I'll go with `min` for duration, `h` for operating hours and `%unit%` for everything else then.